lib/pushmi_pullyu/cli.rb

```diff
@@ -170,7 +170,7 @@ def run_preservation_cycle
           # Log successful preservation event to the log files
           PushmiPullyu::Logging.log_preservation_event(deposited_file)
         end
-      rescue => e
+      rescue StandardError => e
```

Comment:
I would have this one how it was. We want to capture all errors. You can disable this in RoboCop for this one occurrence. By using an inline comment, something like `# rubocop:disable <name of cop error>`

Reply:
OK, this is good point, will change that

Reply:
All changes have been done. Please review again.